assembl/static2/js/app/components/debate/brightMirror/fictionComment.jsx

```diff
@@ -0,0 +1,99 @@
+// @flow
+import React from 'react';
+// Graphql imports
+import { compose, graphql } from 'react-apollo';
+// Helpers imports
+import moment from 'moment';
+// Optimization: Should create commentQuery.graphql and adapt the query
+import CommentQuery from '../../../graphql/BrightMirrorFictionQuery.graphql';
+// HOC imports
+import withLoadingIndicator from '../../../components/common/withLoadingIndicator';
+// Components imports
+import CircleAvatar from './circleAvatar';
+import ToggleCommentButton from '../common/toggleCommentButton';
+import ReplyToCommentButton from '../common/replyToCommentButton';
+// Constant imports
+import { NO_AUTHOR_SPECIFIED, EMPTY_STRING } from '../../../constants';
+// Types imports
+import type { CircleAvatarProps } from './circleAvatar';
+
+// type FictionCommentProps = {};
+
+export type FictionCommentGraphQLProps = {
+  /** Author fullname */
+  authorFullname: string,
+  /** Circle avatar props */
+  circleAvatar: CircleAvatarProps,
+  /** Comment content */
+  commentContent: string,
+  /** Comment displayed published date */
+  displayedPublishedDate: string,
+  /** Number of child comments */
+  numberOfChildComments: number,
+  /** Comment published date, TODO: change it into moment date */
+  publishedDate: string
+};
+
+type LocalFictionCommentProps = FictionCommentGraphQLProps; // & FictionCommentProps;
+
+export const FictionComment = ({
+  authorFullname,
+  circleAvatar,
+  commentContent,
+  displayedPublishedDate,
+  numberOfChildComments,
+  publishedDate
+}: LocalFictionCommentProps) => (
+  <article className="comment-container">
+    <CircleAvatar {...circleAvatar} />
+    <div className="content">
+      <header className="meta">
+        <p>
+          <strong>{authorFullname || NO_AUTHOR_SPECIFIED}</strong>
+        </p>
+        <p className="published-date">
+          <time dateTime={publishedDate} pubdate="true">
+            &nbsp;-&nbsp;{displayedPublishedDate}
+          </time>
+        </p>
+      </header>
+      <p className="comment">{commentContent}</p>
+      <footer className="toolbar">
+        <p>{numberOfChildComments} answers</p>
```

Comment:
This will show "0 answers" if there aren't any. There a few examples in translations.js of the way you can handle 0, 1 and plural for a given word (harvestedExtractNumbers for example)